site/zh-cn/beta/tutorials/keras/basic_text_classification.ipynb

```diff
@@ -0,0 +1,713 @@
+   "metadata": {
+    "colab_type": "text",
+    "id": "RnKvHWW4-lkW"
+   },
+   "source": [
+    "评论文本被转换为整数值，其中每个整数代表词典中的一个单词。首条评论是这样的："
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "QtTS4kpEpjbi"
+   },
+   "outputs": [],
+   "source": [
+    "print(train_data[0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "hIE4l_72x7DP"
+   },
+   "source": [
+    "电影评论可能具有不同的长度。以下代码显示了第一条和第二条评论的中单词数量。由于神经网络的输入必须是统一的长度，我们稍后需要解决这个问题。"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "X-6Ii9Pfx6Nr"
+   },
+   "outputs": [],
+   "source": [
+    "len(train_data[0]), len(train_data[1])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "4wJg2FiYpuoX"
+   },
+   "source": [
+    "### 将整数转换回单词\n",
+    "\n",
+    "了解如何将整数转换回文本对您可能是有帮助的。这里我们将创建一个辅助函数来查询一个包含了整数到字符串映射的字典对象："
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "tr5s_1alpzop"
+   },
+   "outputs": [],
+   "source": [
+    "# 一个映射单词到整数索引的词典\n",
+    "word_index = imdb.get_word_index()\n",
+    "\n",
+    "# 保留第一个索引\n",
+    "word_index = {k:(v+3) for k,v in word_index.items()}\n",
+    "word_index[\"<PAD>\"] = 0\n",
+    "word_index[\"<START>\"] = 1\n",
+    "word_index[\"<UNK>\"] = 2  # unknown\n",
+    "word_index[\"<UNUSED>\"] = 3\n",
+    "\n",
+    "reverse_word_index = dict([(value, key) for (key, value) in word_index.items()])\n",
+    "\n",
+    "def decode_review(text):\n",
+    "    return ' '.join([reverse_word_index.get(i, '?') for i in text])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "U3CNRvEZVppl"
+   },
+   "source": [
+    "现在我们可以使用 `decode_review` 函数来显示首条评论的文本："
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "s_OqxmH6-lkn"
+   },
+   "outputs": [],
+   "source": [
+    "decode_review(train_data[0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "lFP_XKVRp4_S"
+   },
+   "source": [
+    "## 准备数据\n",
+    "\n",
+    "影评——即整数数组必须在输入神经网络之前转换为张量。这种转换可以通过以下两种方式来完成：\n",
+    "\n",
+    "* 将数组转换为表示单词出现与否的由 0 和 1 组成的向量，类似于 one-hot 编码。例如，序列[3, 5]将转换为一个 10,000 维的向量，该向量除了索引为 3 和 5 的位置是 1 以外，其他都为 0。然后，将其作为网络的首层——一个可以处理浮点型向量数据的稠密层。不过，这种方法需要大量的内存，需要一个大小为 `num_words * num_reviews` 的矩阵。\n",
+    "\n",
+    "* 或者，我们可以填充数组来保证输入数据具有相同的长度，然后创建一个大小为 `max_length * num_reviews` 的整型张量。我们可以使用能够处理此形状数据的嵌入层作为网络中的第一层。\n",
+    "\n",
+    "在本教程中，我们将使用第二种方法。\n",
+    "\n",
+    "由于电影评论长度必须相同，我们将使用 [pad_sequences](https://www.tensorflow.org/api_docs/python/tf/keras/preprocessing/sequence/pad_sequences) 函数来使长度标准化："
```

Comment:
```suggestion
    "由于电影评论长度必须相同，我们将使用 [pad_sequences](https://tensorflow.google.cn/api_docs/python/tf/keras/preprocessing/sequence/pad_sequences) 函数来使长度标准化："
```

Reply:
Sorry for these errors, I've made modifications, please check again.